Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -405,7 +410,9 @@ def PyExec(self):
             try:
                 stitched = self.output + "_stitched"
                 Stitch1DMany(InputWorkspaces=outputSamples,
-                             OutputWorkspace=stitched)
+                             OutputWorkspace=stitched,
+                             ScaleRHSWorkspace=True,
+                             indexOfReference=self.stitch_reference_index)
```

Comment:
The key must be start uppercase here.

Reply:
Besides, there is another Stitch at line 460 for the wedges.

Reply:
Currently, they are going to be stitched by default, with the reference being the first one. Would you like to use the same `stitch_reference_index` for both sample data and wedges?

Reply:
Yes.

Reply:
Also, for both cases, make sure the stitch is attempted only if there are more than one workspaces to stitch. Especially now that the default reference index is 1.